tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -1087,6 +1091,47 @@ def distill(self, train_data=None, tuning_data=None, augmentation_data=None, tim
                                      verbosity=verbosity, models_name_suffix=models_name_suffix, teacher_preds=teacher_preds,
                                      augmentation_data=augmentation_data, augment_method=augment_method, augment_args=augment_args)
 
+    def plot_ensemble_model(self):
+        """
+            Output the visualized stack ensemble architecture of a model trained by `fit()`. 
+            The plot is stored to a file, `ensemble_model.png` in folder `Predictor.output_directory` 
+
+            This function requires `graphviz` and `pygraphviz` to be installed because this visualization depends on those package.
```

Comment:
Can you add more detailed install instructions for users (Linux and Mac instructions)?

Reply:
How about having links to graphviz(https://graphviz.org/download/) and pygraphviz(https://pygraphviz.github.io/documentation/pygraphviz-1.6/install.html)? Also, I want to have apt-get install and pip command as an example.